gcp/observability/logging.go

```diff
@@ -329,12 +329,14 @@ func (bml *binaryMethodLogger) buildGCPLoggingEntry(ctx context.Context, c iblog
 			sc := span.SpanContext()
 			gcploggingEntry.Trace = "projects/" + bml.projectID + "/traces/" + sc.TraceID.String()
 			gcploggingEntry.SpanID = sc.SpanID.String()
+			gcploggingEntry.TraceSampled = sc.IsSampled()
```

Comment:
Maybe a little refactor here is in order to reduce duplication?

```go
var sc *SpanContext
if bml.clientSide {
	if span := trace.FromContext(ctx); span != nil {
		sc = span.SpanContext()
	} 
} else {
	sc, _ = opencensus.SpanContextFromContext(ctx)
}
return gcplogging.Entry{
	Timestamp: binLogEntry.GetTimestamp().AsTime(),
	Severity:  100,
	Payload:   grpcLogEntry,
	Trace:       "projects/" + bml.projectID + "/traces/" + sc.TraceID.String(),
	SpanID:    sc.SpanID.String(),
	TraceSampled: sc.IsSampled(),
}
```

Reply:
Ah, right, good idea. The original writes of TraceID/SpanID within the conditional branch was due to the object being different client and server side. Changed to your suggestion.